TestProject/Controllers/Web/WebViewController.m

```diff
@@ -0,0 +1,82 @@
+//
+//  WebViewController.m
+//  TestProject
+//
+//  Created by Developer on 7/20/17.
+//  Copyright © 2017 Developer. All rights reserved.
+//
+
+#import "WebViewController.h"
+
+@interface WebViewController ()
+
+@property (weak, nonatomic) IBOutlet UIWebView *webView;
+@property (weak, nonatomic) IBOutlet UIActivityIndicatorView *activityIndicator;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *backButtonItem;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *forwardButtonItem;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *refreshButtonItem;
+@property (weak, nonatomic) IBOutlet UIBarButtonItem *actionButtonItem;
+
+@end
+
+@implementation WebViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    NSURL *url = [NSURL URLWithString:@"http://apple.com"];
+    NSURLRequest *request = [NSURLRequest requestWithURL:url];
+    [self.webView loadRequest:request];
+    
+}
+
+#pragma mark - Actions
+
+- (IBAction)actionBackButton:(UIBarButtonItem *)sender {
```

Comment:
Template for method name is:
actionSomeButtonClicked

Fix it for all IBActions.